.github/ISSUE_TEMPLATE/support.yaml

```diff
@@ -0,0 +1,108 @@
+name: 🚢 Support
+description: Support needed with Prometheus Operator
+labels: ["kind/support", "needs-triage"]
```

Comment:
Same thing here, does it make sense "needs-triage" for support tickets?

Reply:
Ya this is to just to know if someone from team looked at request yet. This will help us track untriaged ones in report generation later 🙂